.github/workflows/scripts/setup_bundler

```diff
@@ -81,8 +81,7 @@ function install_ruby_version_specific_gems {
     gem install --default digest:3.1.0
     gem install --default digest:3.2.0
     gem install --default erb:4.0.4
-    gem install --default erb:5.0.1
-    gem install --default logger:1.7.0
+    gem install --default erb:5.0.2
```

Comment:
how come this one changed from 5.0.1 to 5.0.2?

Reply:
We were seeing `You have already activated erb 5.0.1, but your Gemfile requires erb 5.0.2.` for some suites!